extensions/sd_api_pictures/script.py

```diff
@@ -105,7 +122,7 @@ def output_modifier(string):
     """
     This function is applied to the model outputs.
     """
-    global pic_id, picture_response, streaming_state
+    global pic_id, picture_response, params
```

Comment:
Would be nice, if there is a way, to skip generating image when loading character which already has some history (now it generates picture, but it is not visible anywhere)

Reply:
Whoa that's a nice catch! Can you list the exact steps to reproduce it?


Gonna think if I can actually override that behaviour